generator/views/funnel_analysis_view.py

```diff
@@ -104,19 +104,22 @@ def _funnel_analysis_lookml(self) -> List[Dict[str, Any]]:
                 "name": f"completed_step_{n}",
                 "type": "yesno",
                 "description": f"Whether the user completed step {n} on the associated day.",
-                "sql": "".join(
+                "sql": "\n".join(
```

Comment:
Might be best to use `dedent` and a multi-line string here instead

Reply:
yes, I'll push that change tomorrow.